src/Microsoft.AspNetCore.Mvc.Core/baseline.netcore.json

```diff
@@ -19488,6 +19488,11 @@
               "Name": "etag",
               "Type": "Microsoft.Net.Http.Headers.EntityTagHeaderValue",
               "DefaultValue": "null"
+            },
+            {
+              "Name": "enableRangeProcessing",
+              "Type": "System.Boolean",
+              "DefaultValue": "True"
```

Comment:
This added parameter is fine. `FileResultExecutorBase` was in `Microsoft.AspNetCore.Mvc.Internal` until 38712609bb and that was post-2.0.